releasePlan.cfg

```diff
@@ -0,0 +1,8 @@
+# 2023 Jan CPU: refer https://www.java.com/releases/
+# variable name in format of "jdk${jdkversion}GA"
+# ${jdkversion} includes "u"
+
+jdk8uGA="jdk8u361"
```

Comment:
This will end in a "2", "1" is from oracles releases...
"jdk8u362"

Reply:
The content of releaseplan.cfg is from what oracels release plan. we no need to predict if it is jdk8u362 or jdk8u363, long need to put what it was announced.
In reality, the GA tag either be the same version number or higher version number.
so, the logic is to find a GA tag, either jdk8u361 or higher than that one (we already know it will be jdk8u362)